### PR TITLE
Quota bugfixes

### PR DESCRIPTION
**Pull request by dannon**
Just took @jmchilton's proposed fix and swap all quota manipulations to use it.  This may fix a race condition which is causing quotas to get out of whack.

### CODE REVIEW

**Comment by dannon** [comment]
(wip because I'm still trying to test and look for other causes -- if anyone else has ideas I'd love to hear 'em)

File: lib/galaxy/model/__init__.py [hunk 1]
```diff
@@ -204,6 +204,10 @@ def set_disk_usage( self, bytes ):
 
     total_disk_usage = property( get_disk_usage, set_disk_usage )
 
+    def adjust_total_disk_usage( self, amount ):
+        from sqlalchemy import func
```
Comment (nsoranzo) [inline lib/galaxy/model/__init__.py]: Can you please move the import at the start of the file?

Comment (dannon) [inline lib/galaxy/model/__init__.py]: Yep.  That said, I'm really more focused on the reproduction and testing at this point -- if you have any ideas about that it'd be super helpful.

Comment (dannon) [inline lib/galaxy/model/__init__.py]: (func was already actually available, this was a duplicate line from testing)

**Comment by nsoranzo** [comment]
:+1: Seems to be the right way to atomically update the value in SQLAlchemy.

**Comment by dannon** [comment]
Dropped WIP.  This seems to work and if I find more quota bugs I'll open a new PR.

**Pull request merged**